.github/workflows/publish-site.yml

```diff
@@ -13,7 +13,13 @@ jobs:
       - name: Install Dependencies
         run: |
           sudo apt-get update -y
-          sudo apt-get install --fix-missing -y lua-ldoc lua-markdown jq p7zip-full libsdl2-2.0-0 libopenal1
+          sudo apt-get install --fix-missing -y jq p7zip-full libsdl2-2.0-0 libopenal1
```

Comment:
@badosu I removed lua-ldoc and lua-markdown, but maybe none of these are needed anymore?

Reply:
Should not be